src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,93 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+interface Props {
+  onAdd: (movie: Movie) => void;
+}
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
```

Comment:
You can use only 1 state for all fields.
Create const outside your component where you can save your `initialFormState` which will have all initial values 

```
const initialFormState = {
  title: '',
  ...
};
```

```suggestion
  const [count, setCount] = useState(0);
  const [formState, setFormState] = useState(initialFromState);
```